### PR TITLE
Add a 'VERSION' file to the SDK build

### PR DESCRIPTION
**Pull request by jstroem**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,6 +42,7 @@ TOITPKG_BIN = $(BIN_DIR)/toitpkg$(EXE_SUFFIX)
 TOITLSP_BIN = $(BIN_DIR)/toitlsp$(EXE_SUFFIX)
 TOITVM_BIN = $(BIN_DIR)/toitvm$(EXE_SUFFIX)
 TOITC_BIN = $(BIN_DIR)/toitc$(EXE_SUFFIX)
+VERSION_FILE = $(BIN_DIR)/VERSION
```
Comment (kasperl) [inline Makefile:45]: Is this automatically packaged up as part of releases?

**Review by kasperl — APPROVED** [review]
Looks good to me!

File: Makefile (hunk 1 above)
Comment (jstroem) [reply Makefile:45]: yes

Comment (jstroem) [reply Makefile:45]: every file in sdk will be available in the tar.

**Pull request merged**